obs/obs.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"os"
+)
+
+type Project struct {
+	XMLName      xml.Name     `yaml:"project" json:"project" xml:"project"`
+	Name         string       `yaml:"name" json:"name" xml:"name,attr"`
+	Title        string       `yaml:"title" json:"title" xml:"title"`
+	Description  string       `yaml:"description" json:"description" xml:"description"`
+	URL          *string      `yaml:"url" json:"url" xml:"url,omitempty"`
+	Persons      []User       `yaml:"person" json:"persons" xml:"person"`
+	Repositories []Repository `yaml:"repositories" json:"repositories" xml:"repository"`
+	Build        *Build       `yaml:"build" json:"build" xml:"build,omitempty"`
+	Publish      *Publish     `yaml:"publish" json:"publish" xml:"publish,omitempty"`
+	DebugInfo    *DebugInfo   `yaml:"debugInfo" json:"debugInfo" xml:"debuginfo,omitempty"`
+	UseForBuild  *UseForBuild `yaml:"useForBuild" json:"useForBuild" xml:"useforbuild,omitempty"`
+	Parent       *string      `yaml:"parent" json:"parent" xml:"parent"`
+}
+
+type Client struct {
+	Username string
+	Password string
+}
+
+type Path struct {
+	Project    string `xml:"project,attr"`
+	Repository string `xml:"repository,attr"`
+}
+
+type Build struct {
+	Disable string `xml:"disable"`
+}
+
+type Publish struct {
+	Disable string `xml:"disable"`
+}
+
+type DebugInfo struct {
+	Disable string `xml:"disable"`
+}
+
+type UseForBuild struct {
+	Disable string `xml:"disable"`
+}
+
+type User struct {
+	UserID string `xml:"userid,omitempty"`
+	Role   role   `xml:"role,omitempty"`
+}
+
+type Repository struct {
+	Name         string         `xml:"name,attr"`
+	Architecture []string       `xml:"arch"`
+	RT           *ReleaseTarget `xml:"releasetarget,omitempty"`
```

Comment:
- There can be multiple release targets so this should be a slice
- Let's please use a full name, maybe ReleaseTargets